site/src/pages/WorkspacePage/Workspace.tsx

```diff
@@ -257,6 +282,53 @@ export const Workspace: FC<WorkspaceProps> = ({
   );
 };
 
+const WorkspaceResourceData: FC<{ resource: TypesGen.WorkspaceResource }> = ({
+  resource,
+}) => {
+  const metadata = resource.metadata ?? [];
+
+  if (resource.daily_cost > 0) {
+    metadata.push({
+      key: "Daily cost",
+      value: resource.daily_cost.toString(),
+      sensitive: false,
+    });
+  }
+
+  if (metadata.length === 0) {
+    return null;
+  }
+
+  return (
+    <header css={styles.resourceData}>
+      {resource.metadata?.map((meta) => {
+        return (
+          <div css={styles.resourceDataItem} key={meta.key}>
+            <div css={styles.resourceDataItemValue}>
+              {meta.sensitive ? (
+                <SensitiveValue value={meta.value} />
+              ) : (
+                <MemoizedInlineMarkdown components={{ p }}>
+                  {meta.value}
+                </MemoizedInlineMarkdown>
+              )}
+            </div>
+            <div css={styles.resourceDataItemLabel}>{meta.key}</div>
+          </div>
+        );
+      })}
+    </header>
+  );
+};
+
+const p = ({ children }: PropsWithChildren) => {
+  const childrens = Children.toArray(children);
+  if (childrens.every((child) => typeof child === "string")) {
+    return <CopyableValue value={childrens.join("")}>{children}</CopyableValue>;
+  }
+  return <>{children}</>;
```

Comment:
What's going on here?

Reply:
Hm... I think it is some copyable value wrapper 🤔 git, or GH is not doing a good work on isolating/identifying only the changes that I made... but I will see how we can improve this.